### PR TITLE
Decomp 2F300.c

### PR DESCRIPTION
**Pull request by KiritoDv**
- func_8002EE34
- func_8002F64C
- func_8002FE74

### CODE REVIEW

**Comment by sonicdcer** [comment]
> * func_8002EE34

Hello and welcome to SF64 Decomp! Thank you for your contribution. Can you do a make format to format the code please? :)

File: src/main/sf_2F300.c [hunk 1]
```diff
@@ -18,7 +20,12 @@ void func_8002EE34(void) {
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_2F300/func_8002F5F4.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_2F300/func_8002F64C.s")
+void func_8002F64C(void) {
+    if ((D_80177CD0[0x0] + D_80177CD0[0x1] + D_80177CD0[0x2] + D_80177CD0[0x3]) == 0) {
```
Comment (sonicdcer) [inline src/main/sf_2F300.c:24]:
```suggestion
    if ((D_80177CD0[0] + D_80177CD0[1] + D_80177CD0[2] + D_80177CD0[3]) == 0) {
```

Comment (sonicdcer) [reply src/main/sf_2F300.c:24]: Arrays in DEC i think would be better :)

**Review by sonicdcer — APPROVED** [review]

File: src/main/sf_2F300.c [hunk 2]
```diff
@@ -1,10 +1,16 @@
-#include "common.h"
+#include "global.h"
+
+extern s32 D_8015F904;
+extern s32 D_80177CD0[12];
```
Comment (petrie911) [inline src/main/sf_2F300.c:4]: Are you sure this has size 12? We've only seen four elements of it in the file.

**Pull request merged**